src/fu-engine.h

```diff
@@ -35,7 +35,7 @@ G_BEGIN_DECLS
 #define FU_TYPE_ENGINE (fu_engine_get_type ())
 G_DECLARE_FINAL_TYPE (FuEngine, fu_engine, FU, ENGINE, GObject)
 
-#define		FU_ENGINE_FIRMWARE_SIZE_MAX		(32 * 1024 * 1024) /* bytes */
+#define		FU_ENGINE_FIRMWARE_SIZE_MAX		(512 * 0x100000) /* bytes * Mb */
```

Comment:
Why keep around both `FU_ENGINE_FIRMWARE_SIZE_MAX `and `FU_COMMON_ARCHIVE_SIZE_MAX`?  It seems like you should be able to get away with 1 or the other.  I know they're set to different values right now, but so why?
If both should stick around a comment around their intent might be useful for the future.

Reply:
I pondered about moving the define to fu-common-cab.h, but you're right, we can just use the old define name. Fixed.